### PR TITLE
small changes to get dockernet integration tests running as consumer chain

### PR DESCRIPTION
**Pull request by sampocs**
* Switched from hermes to relayer
* Removed hermes setup
* Changed init_chain to work with any number of stride nodes

### CODE REVIEW

File: dockernet/src/init_chain.sh [hunk 1]
```diff
@@ -139,12 +137,14 @@ for (( i=1; i <= $NUM_NODES; i++ )); do
...
-        cp $DOCKERNET_HOME/state/${STRIDE_NODE_PREFIX}${i}/config/node_key.json $DOCKERNET_HOME/state/${NODE_PREFIX}${i}/config/node_key.json
+    if [[ $CHAIN == "GAIA" ]]; then
+        stride_config=$DOCKERNET_HOME/state/${STRIDE_NODE_PREFIX}${i}/config
+        host_config=$DOCKERNET_HOME/state/${NODE_PREFIX}${i}/config
+        cp ${stride_config}/priv_validator_key.json ${host_config}/priv_validator_key.json
+        cp ${stride_config}/node_key.json ${host_config}/node_key.json
     fi
 
-    if [[ ($CHAIN == "STRIDE" && ($i == 1 || $i == 2)) || $CHAIN != "STRIDE" ]]; then
+    if [[ ($CHAIN == "STRIDE" && ($i != 0)) || $CHAIN != "STRIDE" ]]; then
```
Comment (strbrian) [inline dockernet/src/init_chain.sh:147]: The index `i` of this loop actually starts from 1. 
```suggestion
    if [[ ($CHAIN == "STRIDE" && ($i <= 2)) || $CHAIN != "STRIDE" ]]; then
```

**Review by strbrian — CHANGES_REQUESTED** [review]
Overall looking great.
One thing should be addressed before merging

File: dockernet/src/init_chain.sh (hunk 1 above)
Comment (sampocs) [reply dockernet/src/init_chain.sh]: chatted offline, decided to remove this if statement

**Review by strbrian — APPROVED** [review]
Nice!

**Pull request merged**